spring-cloud-alibaba-starters/spring-cloud-starter-alibaba-nacos-discovery/src/main/java/com/alibaba/cloud/nacos/util/InetIPv6Utils.java

```diff
@@ -39,64 +33,56 @@
 /**
  * @author HH
  */
-public class InetIPv6Utils implements Closeable {
+public class InetIPv6Utils {
 
-	private final ExecutorService executorService;
+	private final static Log log = LogFactory.getLog(InetIPv6Utils.class);
 
-	private final Log log = LogFactory.getLog(InetIPv6Utils.class);
+	private final static String IPV6_IS_NULL = "";
```

Comment:
Don't define the final field, use Strings.EMPTY.

Reply:
StringUtils.EMPTY